xml/book_sles_docker.xml

```diff
@@ -36,7 +36,7 @@
  <!-- Installation of Docker -->
  <xi:include href="docker_installation.xml"/>
  <!-- Installation of sle2docker -->
- <xi:include href="docker_sle2docker.xml"/>
+ <!--<xi:include href="docker_sle2docker.xml"/>-->
```

Comment:
As the tool is removed -- why not just cleanly remove the file instead of commenting this content? If we need to restore, there is always Git.

Reply:
Done with https://github.com/SUSE/doc-sle/pull/321/commits/6d90d1eec137250563 (just was waiting for tech review before removing completely).